src/middleware.js

```diff
@@ -15,6 +15,7 @@ function isAuthenticated() {
 }
 
 // Determine whether the authorization options passed to isAuthorized are valid.
+//eslint-disable-next-line no-unused-vars
```

Comment:
What is it complaining about here?  Can we fix the issue instead?

Reply:
There is this function which is unused, causing eslint to throw `no-unused-vars`:
```
function validateAuthorizationOptions(options = {}) { ... }
```
So, I added the ignore statement. Now while, adding it to the exports would fix this problem, I wasn't sure if it should be exposed. Please let me know if that is more ideal:
```
module.exports.validateAuthorizationOptions= validateAuthorizationOptions;
```

Reply:
This is a good example of where it's better to listen to eslint than ignore it.  In the past, we used to allow passing an Object to `isAuthorized()`, and we would validate it, see https://github.com/Seneca-CDOT/satellite/pull/8/files#diff-9a117d52a8a2cb3e5f498575bdfdea4dc32418180c09029454fbaf7f95ebe5a1R42.  However, it looks like we dropped that, and now require you to pass a function instead.  As a result, the right fix here is to delete this entire funcction, because it's not being used anymore.  We don't want to keep code that isn't being used.

Reply:
Didn't know about that, thanks for the insight. Removed with the latest changes.